.appveyor.yml

```diff
@@ -13,6 +13,7 @@ branches:
 # build cache to preserve files/folders between builds
 cache:
   - dependencies.7z
+  # - qt-5.12.1-msvc.7z # if not using AppVeyor's built-in Qt in before_build.bat
   - C:\Qt\Tools\mingw530_32\lib
   - C:\Qt\Tools\mingw530_32\i686-w64-mingw32
```

Comment:
I don't think these 2 are needed?

Reply:
Probably not, but they are not related to this PR and I don't have time to investigate other things right now.